psi4/src/psi4/libscf_solver/hf.cc

```diff
@@ -328,6 +330,13 @@ void HF::common_init()
         potential_ = nullptr;
     }
 
+    // post-scf vv10 correlation
+    if(options_.get_bool("DFT_VV10_POSTSCF")) {
```

Comment:
Can you format these blocks? We normally use clang-format, but basically add spaces around operations, `if(` should be `if (`, and the code block should be indented by 4 spaces.